commands/tweet.js

```diff
@@ -0,0 +1,33 @@
+var TwitterClient = require('../libs/twitter-client-instance');
+
+function getLatestStatus(tweets) {
+    return tweets.statuses[0];
+}
+
+module.exports = function (bb8) {
+    console.log('Let\'s Get Tweets!!');
+
+    var executeTwitterCommand = function () {
+        TwitterClient.get('search/tweets', {q: 'bb8bbc'}, function (error, tweets) {
```

Comment:
Yeah, was thinking that... Just ran out of time to play with it last night... I'll make the change later :)